### DISCUSSION TITLE
PUT request with edge

### DISCUSSION THREAD

**Discussion by subhamcodes** [adonisjs/core#1293] [Help] [answered]
I want to submit a PUT request but the browser returns **E_ROUTE_NOT_FOUND: Cannot POST:/posts/1** 

    <form 
         action="{{ route('PostsController.update', { id: post.id }) }}?_method=PUT" 
         method="POST" 
         enctype="multipart/form-data"
    >

In the routes.js i have the following code

    Route.put('/posts/:id', 'PostsController.update').middleware('auth')

**Comment by McSneaky** [comment]
Set your form method to `PUT` :wink:

**Reply by thetutlage** [reply]
Won't work. Since native forms only support POST and GET 🙃

**Comment by thetutlage** [comment, marked as answer]
You need to enable form method spoofing inside config/app.ts file

**Reply by subhamcodes** [reply]
Thanks. It worked like charm
